src/main/res/values/strings.xml

```diff
@@ -23,7 +23,7 @@
     <!-- First Run Dialog -->
 
     <string name="firstrun_login_title">Log in</string>
-    <string name="firstrun_login_message">You are not currently logged in. You can access the account list by selecting "Accounts" from the menu.</string>
+    <string name="firstrun_login_message">You are not logged in. Access the account list by selecting \"Accounts\" from the menu.</string>
```

Comment:
I prefer the original here. The wording is deliberately "you can" because the first-run dialog already contains a button to log in, so the message is just talking about a future possibility rather than something the user should do now. Also prefer the first sentence with "currently".